### PR TITLE
GPU support docs

### PR DESCRIPTION
**Pull request by sascala**
## What type of changes does your doc introduce?
- [ ] Bug fix
- [X] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [X] High
- [ ] Medium

## I have completed these items:
- [ ] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Comment by sascala** [comment]
@klueska The page is a barebones GPU document. Any thoughts you have on what else a user might need / want to know are very welcome.

File: 1.8/usage/gpu-support.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+You can configure your DC/OS service to use the GPUs (Graphics Processing Units) of your nodes. GPUs can dramatically accelerate big data workloads. [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Add the `enable_gpu_isolation: true` flag to your `config.yaml` when you install DC/OS on your cluster. If you are installing DC/OS from the AWS cloud template, follow [the instructions here](/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/). Otherwise, follow [the custom installation instructions](/docs/1.8/administration/installing/custom/).
+
+1. Ensure that all machines in your cluster have [Nvidia GPU libraries](https://developer.nvidia.com/gpu-accelerated-libraries) installed on them. The machines do not need to have GPUs on them, but they will fail to start without the libraries.
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: This is the library they need to have installed: https://developer.nvidia.com/nvidia-management-library-nvml

I would provide a link to that instead.

File: 1.8/usage/gpu-support.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+post_title: GPU Acceleration
+nav_title: GPU Acceleration
+feature_maturity: experimental
+menu_order: 110 
+---
+
+You can configure your DC/OS service to use the GPUs (Graphics Processing Units) of your nodes. GPUs can dramatically accelerate big data workloads. [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: For some reason the first sentence here feels unnatural. Saying "the GPUs of your nodes" sounds a little off.

I would maybe reword it as:
```
DC/OS now supports allocating GPUs (Graphics Processing Units) to your long running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads.  [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).

By default, DC/OS is not configured to run with GPUs, so you must explicitly enable this support when first installing DC/OS on your cluster.
```

File: 1.8/usage/gpu-support.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Add the `enable_gpu_isolation: true` flag to your `config.yaml` when you install DC/OS on your cluster. If you are installing DC/OS from the AWS cloud template, follow [the instructions here](/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/). Otherwise, follow [the custom installation instructions](/docs/1.8/administration/installing/custom/).
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: I would reverse these two steps. They should first ensure that they have the libraries installed, and then add the configuration flag to `config.yaml` and install DC/OS on their cluster.

File: 1.8/usage/gpu-support.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+     "mem": 128,
+     "disk": 0,
+     "gpus": 1,
+     "instances": 1
+}
+```
+
+## Docker-Based Application Definition
+
+**Important:** You must use the [DC/OS Universal Container Runtime](/docs/1.8/usage/containerizers/) to run a containerized application that uses GPUs.
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: I would mention that this is ensured in the example below by setting the container type to "MESOS".

File: 1.8/usage/gpu-support.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+     "id": "simple-gpu-test",
+     "acceptedResourceRoles":["slave_public", "*"],
+     "cmd": "while [ true ] ; do nvidia-smi; sleep 5; done",
+     "cpus": 1,
+     "mem": 128,
+     "disk": 0,
+     "gpus": 1,
+     "instances": 1
+}
+```
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: I would probably mention here how to navigate to the logs to ensure that the output of this command runs successfully and shows the proper output from the nvidia-smi command. The output should look something like the following repeated once every 5 seconds:

```
+------------------------------------------------------+
| NVIDIA-SMI 352.79     Driver Version: 352.79         |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Tesla M60           Off  | 0000:04:00.0     Off |                    0 |
| N/A   34C    P0    39W / 150W |     34MiB /  7679MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
```

Maybe follow the GPU documentation I wrote for Mesos as a template:
https://github.com/apache/mesos/blob/master/docs/gpu-support.md

File: 1.8/usage/gpu-support.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    "instances": 1,
+    "container": {
+      "type": "MESOS",
+      "docker": {
+        "network": "HOST",
+        "image": "nvidia/cuda"
+      }
+    }
+}
+```
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: Ditto on the verifying output that I mentioned above.

File: 1.8/usage/gpu-support.md (hunk 1 above)
Comment (klueska) [inline 1.8/usage/gpu-support.md]: Oh, I also have detailed instructions on how to install them here:
https://github.com/apache/mesos/blob/master/docs/gpu-support.md#external-dependencies

If they are running on AWS though, they can simply use one of these AMIs instead of the default DC/OS one:
```
us-west-2: ami-9b5d97fb
us-east-1: ami-e10e50f6
ap-southeast-2: ami-37b28f54
```

File: 1.8/usage/gpu-support.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+You can configure your DC/OS service to use the GPUs (Graphics Processing Units) of your nodes. GPUs can dramatically accelerate big data workloads. [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Add the `enable_gpu_isolation: true` flag to your `config.yaml` when you install DC/OS on your cluster. If you are installing DC/OS from the AWS cloud template, follow [the instructions here](/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/). Otherwise, follow [the custom installation instructions](/docs/1.8/administration/installing/custom/).
+
+1. Ensure that all machines in your cluster have [Nvidia GPU libraries](https://developer.nvidia.com/gpu-accelerated-libraries) installed on them. The machines do not need to have GPUs on them, but they will fail to start without the libraries.
+
```
Comment (klueska) [inline 1.8/usage/gpu-support.md]: They specifically want to install the NVML library. Maybe just change the text to:
```
Ensure that all machines in your cluster have the [Nvidia Management Library (NVML)](https://developer.nvidia.com/nvidia-management-library-nvml) installed on them. The machines do not need to have GPUs on them, but they will fail to to come online without this library.
```

Comment (sascala) [inline 1.8/usage/gpu-support.md]: 👍

File: 1.9/usage/gpu-support.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
+---
+post_title: GPU Acceleration
+nav_title: GPU Acceleration
+feature_maturity: experimental
+menu_order: 110 
+---
+
+DC/OS now supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads.  [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+By default, DC/OS is not configured to run with GPUs, so you must explicitly enable this support when first installing DC/OS on your cluster. Once configured, DC/OS treats  GPUs as another resource available to your app.
```
Comment (klueska) [inline 1.9/usage/gpu-support.md]: We should replace the term "app" here with "service".

File: 1.9/usage/gpu-support.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Ensure that all machines in your cluster have the [Nvidia Management Library (NVML)](https://developer.nvidia.com/nvidia-management-library-nvml) installed on them. The machines do not need to have GPUs on them, but they will fail to to come online without this library.
```
Comment (klueska) [inline 1.9/usage/gpu-support.md]: I know aIl already commented on this line, but maybe we should be a bit more clear in the last sentence, maybe:
```
The machines do not need to have GPUs on them, but they will fail to to come online without this library if GPU support is enabled (as described below).
```

File: 1.9/usage/gpu-support.md (hunk 8 above)
Comment (sascala) [inline 1.9/usage/gpu-support.md]: indeed

File: 1.9/usage/gpu-support.md (hunk 9 above)
Comment (sascala) [inline 1.9/usage/gpu-support.md]: 👍

**Comment by sascala** [comment]
Thanks for your careful eye, @klueska

File: 1.9/usage/gpu-support.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|===============================+======================+======================|
+|   0  Tesla M60           Off  | 0000:04:00.0     Off |                    0 |
+| N/A   34C    P0    39W / 150W |     34MiB /  7679MiB |      0%      Default |
++-------------------------------+----------------------+----------------------+
+```
+
+## Docker-Based Application Definition
+
+**Important:** You must use the [DC/OS Universal Container Runtime](/docs/1.8/usage/containerizers/) to run a containerized application that uses GPUs. To use the Universal Containerizer, set the container type to `MESOS`.
```
Comment (klueska) [inline 1.9/usage/gpu-support.md]: Should be `To use the Universal Container Runtime ...`

Comment (sascala) [inline 1.9/usage/gpu-support.md]: thanks for catching that

**Comment by sascala** [comment]
@pyronicide If you're the right PM for this, could you take a quick look?

File: 1.8/usage/gpu-support.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+You can configure your DC/OS service to use the GPUs (Graphics Processing Units) of your nodes. GPUs can dramatically accelerate big data workloads. [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Add the `enable_gpu_isolation: true` flag to your `config.yaml` when you install DC/OS on your cluster. If you are installing DC/OS from the AWS cloud template, follow [the instructions here](/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/). Otherwise, follow [the custom installation instructions](/docs/1.8/administration/installing/custom/).
+
```
Comment (joel-hamill) [inline 1.8/usage/gpu-support.md]: Nope, this *should* work, based on my understanding.

**Comment by joel-hamill** [comment]
Related eng PR https://github.com/dcos/dcos/pull/766

File: 1.9/usage/gpu-support.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+        enable_gpu_isolation: true
+        aws_template_storage_bucket: <s3-bucket-name>
+        aws_template_storage_bucket_path: templates/dcos
+        aws_template_upload: true
+        aws_template_storage_access_key_id: <your-access-key-id>
+        aws_template_storage_secret_access_key: <your-secret-access_key>
+
+1. When you create your stack on [CloudFormation](https://console.aws.amazon.com/cloudformation/home), choose one of these three regions: US West (Oregon), US East (N. Virginia), or Asia Pacific (Sydney).
+
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: All of these parameters must be specified: ```   CustomAmi: ami-9b5d97fb (this is our standard CentOS image with Nvidia GPU tools installed)
   InternetGateway: <custom output from zen>
   # This is the required AWS machine type for GPUs.
   PrivateAgentInstanceType: g2.2xlarge
   PrivateSubnet: <custom output from zen>
   # This is the required AWS machine type for GPUs.
   PublicAgentInstanceType: g2.2xlarge
   PublicSubnet: <custom output from zen>
   Vpc: vpc-1a5f437e <custom output from zen>```

**Comment by sascala** [comment]
blocked by https://issues.apache.org/jira/browse/MESOS-7027

File: 1.9/usage/gpu-support.md [hunk 13]
```diff
@@ -0,0 +1,224 @@
+---
+post_title: GPU Acceleration
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: This title is a bit ambiguous

File: 1.9/usage/gpu-support.md [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+
+1. On the **Select Template** page, specify the custom template that you created on your master node via the `sudo bash dcos_generate_config.sh --aws-cloudformation` command . For example, `
+https://s3-us-west-2.amazonaws.com/joel-test-gpu/templates/dcos/config_id/fe4f3d47e23eda85301c875a28a34a8f48e44f95/cloudformation/el7-zen-1.json`.
+
+1. On the **Specify Details** page, specify these values and and click **Next**.
+    - **StackName** - Specify your cluster name.
+    - **AdminLocation** - Accept default.
+    - **CustomAMI** - Specify the custom AMI for your region:
+
+      us-west-2: ami-9b5d97fb
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: Should make this a bulleted list.

Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: Make the AMI names command syntax (e.g. `ami-9b5d97fb`).

File: 1.9/usage/gpu-support.md [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+      us-east-1: ami-e10e50f6
+      ap-southeast-2: ami-37b28f54
+
+    - **InternetGateway** - Specify the InternetGatewayId value from your zen template dependencies (e.g. igw-f15f0895).
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: Make value command sytnax: `igw-f15f0895`

File: 1.9/usage/gpu-support.md [hunk 16]
```diff
@@ -0,0 +1,224 @@
...
+1. Use these dependency values as input to create your stack in CloudFormation in the next steps.
+
+#### Deploy the CF template
+
+1. Go to [CloudFormation](https://console.aws.amazon.com/cloudformation/home) and click **Create Stack**.
+
+1. On the **Select Template** page, specify the custom template that you created on your master node via the `sudo bash dcos_generate_config.sh --aws-cloudformation` command . For example, `
+https://s3-us-west-2.amazonaws.com/joel-test-gpu/templates/dcos/config_id/fe4f3d47e23eda85301c875a28a34a8f48e44f95/cloudformation/el7-zen-1.json`.
+
+1. On the **Specify Details** page, specify these values and and click **Next**.
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: To make this work for both OSS and ENT i think you could reword to say: `On the **Specify Details** page, specify your stack information and click **Next**. Here are the GPU-specific settings.` - and then don't include stuff that isn't GPU specific (e.g. oauth, PublicAgentInstanceCount). 

Also, typo `and and click`.

File: 1.9/usage/gpu-support.md [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+    - **OAuthEnabled** - Indicate whether to enable OAuth.
+    - **PrivateAgentInstanceCount** - Specify the number of private agent nodes.
+    - **PrivateAgentInstanceType** - Specify a machine type of `g2.2xlarge`. This is the GPU-supported machine type.
+    - **PrivateSubnet** - Specify the Private SubnetId value from your zen template dependencies (e.g. subnet-e7193fbf).
+    - **PublicAgentInstanceCount** - Specify the number of public agent nodes.
+    - **PublicAgentInstanceType** - Specify a machine type of `g2.2xlarge`. This is the GPU-supported machine type.
+    - **PublicSubnet** - Specify the Public SubnetId value from your zen template dependencies (e.g. subnet-98193fc0).
+    - **Vpc** - Specify the VpcId value from your zen template dependencies (e.g. vpc-49a0202e).
+
+1. On the Options page, accept the defaults and click Next.
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: Make UI elements bold (e.g. ** Options**).

File: 1.9/usage/gpu-support.md [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+    **Tip**: You can choose whether to rollback on failure. By default this option is set to Yes.
+
+1. On the Review page, check the acknowledgement box, then click **Create**.
+    **Tip**: If the **Create New Stack** page is shown, either AWS is still processing your request or you’re looking at a different region. Navigate to the correct region and refresh the page to see your stack.
```
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: Add line break for tip.

File: 1.9/usage/gpu-support.md [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+
+## Configure a Non-AWS Cluster
```
Comment (klueska) [inline 1.9/usage/gpu-support.md]: Can we maybe put the Non-AWS instructions first and maybe stress how trivial it actually is to enable GPU support (just a single added configuration parameter and the installation of a library dependency).

All of the complication we see in the instructions comes from the AWS specific confiration / installation that needs to be done, and I think it distracts from what is really going on.  It's necessary to have those instructions here (rather than simply linking to them), but I think the way things are organized currently distracts from what is really necessary just to get GPU support working.

Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: Maybe we can break this out into 3 separate child topics?
# Enabling GPU support
## Configure Cluster
### Configuring your AWS Cluster
### Configuring your non-AWS Cluster
## Configure App Definition

**Comment by sascala** [comment]
That's a good point, @klueska, and a good suggestion, @joel-hamill. What do you think of the topic now?

**Comment by joel-hamill** [comment]
@sascala please run a local build and verify the formatting - some things are messed up (e.g. https://cl.ly/2W0R2l0N3Y0d).

File: 1.9/usage/gpu-support.md (hunk 14 above)
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: @sascala i don't see this change implemented in the latest.

File: 1.9/usage/gpu-support.md (hunk 16 above)
Comment (joel-hamill) [inline 1.9/usage/gpu-support.md]: @sascala all of the parameters are still shown https://cl.ly/1Z2M0J2t3v0o? Please see my suggestion above.

**Comment by sascala** [comment]
better, @joel-hamill ?

File: 1.9/usage/gpu-support.md (hunk 14 above)
Comment (sascala) [inline 1.9/usage/gpu-support.md]: oh sorry - misunderstood what you wanted there

File: 1.9/usage/gpu-support.md (hunk 16 above)
Comment (sascala) [inline 1.9/usage/gpu-support.md]: slipped through the cracks -- fixing...

**Comment by sascala** [comment]
@mesosphere/documentation-team testing testing

File: 1.9/usage/gpu/index.md [hunk 20]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: @klueska @sascala should we add a link for further reading to this presentation: https://docs.google.com/presentation/d/1FnuEW2ic5d-cpSyVOUMfUSM7WxJlZtTAAWt2dZXJ52A/edit#slide=id.g166a287193_0_42

**Comment by klueska** [comment]
If you're adding links, I also wrote this Mesos-specific documentation:
https://github.com/apache/mesos/blob/master/docs/gpu-support.md

And have a couple other relevant presentations to link:
https://www.youtube.com/watch?v=giJ4GXFoeuA
https://docs.google.com/presentation/d/1Y1IUlWV6g1HzD1wYIYXy6AmbfnczWfjvvmqqpeDFBic
https://docs.google.com/presentation/d/1FnuEW2ic5d-cpSyVOUMfUSM7WxJlZtTAAWt2dZXJ52A

File: 1.9/usage/gpu/limitations.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (sascala) [inline 1.9/usage/gpu/limitations.md]: @klueska Does this look OK to you? Anything you'd add?

File: 1.9/usage/gpu/aws-install.md [hunk 22]
```diff
@@ -0,0 +1,39 @@
...
+
+## Create Dependencies
+
+1. Run the `zen.sh` script to create the Zen template dependencies. These dependencies will be used as input to create your stack in CloudFormation.
+  ```
+  bash ./zen.sh <stack-name>
+  ```
+  **Important:** You must run the `zen.sh` script before performing the next steps.
+
+1. Follow the instructions [here](docs/1.9/administration/installing/cloud/aws/advanced/quickstart/) to create a cluster with advanced AWS templates, but use the following GPU-specific configuration.
```
Comment (joel-hamill) [inline 1.9/usage/gpu/aws-install.md]: Global: relative links must have the forward slash`/`, as in `(/docs/1.9/administration/installing/cloud/)`

File: 1.9/usage/gpu/index.md [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+post_title: Enable GPU Acceleration
+nav_title: Enable GPU Acceleration
+feature_maturity: experimental
+menu_order: 110
+---
+
+DC/OS now supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads.  [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: Suggest `DC/OS supports allocating...` - the word `now` won't make much sense as new versions are released.

Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: Can the "Learn more" link be moved to the bottom of the page with the others? We should avoid sending people off of the page in the first paragraph.

File: 1.9/usage/gpu/index.md [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+
+# Configure your cluster for GPUs
+
+Follow the steps in this topic to enable GPU support on your cluster. We provide instructions for both AWS cloud and custom installations.
+
+**Important:** All machines in your cluster that have GPUs must have the [Nvidia Management Library (NVML)](https://developer.nvidia.com/nvidia-management-library-nvml) installed on them.
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: Move this paragraph to the individual instruction sections (/custom-install/ and /aws-install/), as a prerequisite.

File: 1.9/usage/gpu/limitations.md [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+feature_maturity: experimental
+menu_order: 3
+---
+
+- Unlike other resources, like  CPUs, memory, and disk, you can only specify whole numbers of GPUs in your application definition. If a fractional amount is selected, launching the task will result in a `TASK_ERROR`.
+
+- Nvidia GPU support is only available for tasks launched through the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/). No support exists for launching GPU-capable tasks through the Docker containerizer. However, the DC/OS Universal container runtime supports running Docker images natively.
+
+- While GPU resources are advertised to the Mesos master alongside other resources like CPUs, memory, and disk, the master will only forward offers that contain GPUs to DC/OS services that have explicitly enabled the GPU_RESOURCES capability. Below is an example of setting this capability in a C++-based service.
+  ```c++
```
Comment (joel-hamill) [inline 1.9/usage/gpu/limitations.md]: This code is not formatted properly - renders like this: https://cl.ly/331u2L2V2h3o.

File: 1.9/usage/gpu/index.md [hunk 26]
```diff
@@ -0,0 +1,21 @@
...
+
+## Further reading
+
+- [Mesos-specific documentation](https://github.com/apache/mesos/blob/master/docs/gpu-support.md)
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: Why would DC/OS users want to read this? Can we change to `Mesos Nvidia GPU Support`

CC: @klueska

File: 1.9/usage/gpu/index.md [hunk 27]
```diff
@@ -0,0 +1,21 @@
+---
+post_title: Enable GPU Acceleration
+nav_title: Enable GPU Acceleration
+feature_maturity: experimental
+menu_order: 110
+---
+
+DC/OS now supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads.  [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your cluster for GPUs
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: Remove this heading since the instructions aren't actually here. Replace with something like `To get started, create a GPU-enabled DC/OS cluster.` And then add hyperlink headings to both sets of instructions. The current topic makes it unclear what the next steps are - the further reading links are most-obvious, but probably least applicable to getting started.

File: 1.9/usage/gpu/custom-install.md [hunk 28]
```diff
@@ -0,0 +1,12 @@
+---
+post_title: Configure Your Cluster
+nav_title: Configure Your Cluster
+feature_maturity: experimental
+menu_order: 1
+---
+
```
Comment (joel-hamill) [inline 1.9/usage/gpu/custom-install.md]: This topic needs an introduction and some context. What is a "custom" installation? Minimally  a hyperlink back to dcos/io/docs/1.9/administration/installing/.

File: 1.9/usage/gpu/gpu-app-definition.md [hunk 29]
```diff
@@ -0,0 +1,76 @@
+---
+post_title: Configure Your Service
+nav_title: Configure Your Service for GPUs
```
Comment (joel-hamill) [inline 1.9/usage/gpu/gpu-app-definition.md]: The purpose of nav_title in dcos.io is generally for a shorter versions of a title, not longer.

File: 1.9/usage/gpu/index.md [hunk 30]
```diff
@@ -0,0 +1,21 @@
+---
+post_title: Enable GPU Acceleration
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: `nav_title` is optional (for a shorter version of a title which appears in the sidebar). it shouldn't be a duplicate of post_title - you could just leave out in this case.

File: 1.9/usage/gpu/index.md [hunk 31]
```diff
@@ -0,0 +1,21 @@
...
+nav_title: Enable GPU Acceleration
```
Comment (joel-hamill) [inline 1.9/usage/gpu/index.md]: Title should be a gerund - e.g. `Enabling GPU Acceleration`

File: 1.9/usage/gpu/custom-install.md [hunk 32]
```diff
@@ -0,0 +1,12 @@
+---
+post_title: Configure a non-AWS Cluster
```
Comment (joel-hamill) [inline 1.9/usage/gpu/custom-install.md]: Diff capitalization of `Non`
